content-src/components/DiscoveryStreamComponents/DSLinkMenu/DSLinkMenu.jsx

```diff
@@ -43,18 +43,14 @@ export class _DSLinkMenu extends React.PureComponent {
     const {index, dispatch} = this.props;
     const isContextMenuOpen = this.state.showContextMenu && this.state.activeCard === index;
     const TOP_STORIES_CONTEXT_MENU_OPTIONS = ["CheckBookmarkOrArchive", "CheckSavedToPocket", "Separator", "OpenInNewWindow", "OpenInPrivateWindow", "Separator", "BlockUrl"];
-    const title = this.props.title || this.props.source;
     const type = this.props.type || "DISCOVERY_STREAM";
 
     return (<div>
       <button ref={this.contextMenuButtonRef}
               className="context-menu-button icon"
               title={this.props.intl.formatMessage({id: "context_menu_title"})}
-              onClick={this.onMenuButtonClick}>
-        <span className="sr-only">
-          <FormattedMessage id="context_menu_button_sr" values={{title}} />
-        </span>
-      </button>
+              aria-label={this.props.intl.formatMessage({id: "context_menu_title"})}
```

Comment:
Is this an intended change from `context_menu_button_sr` to `context_menu_title` ?